### PR TITLE
:bookmark: release 0.8.1

### PR DESCRIPTION
**Pull request by annashamray**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
 =========
 
+0.8.1 (2024-09-25)
+------------------
+
+**Bugfixes/QOL**
+
+* Change SESSION_COOKIE_SAMESITE to "Lax" to fix OIDC (#72)
+* Remove url from SECRET_KEY help text (#76)
+
```
Comment (Coperh) [inline CHANGELOG.rst:11]:
```suggestion
* Change SESSION_COOKIE_SAMESITE to "Lax" to fix OIDC (#72)
* Remove url from SECRET_KEY help text (#76)
* Change CSP headers to support API schema page

```

Can you the CSP headers change before releasing?

**Review by Coperh — APPROVED** [review]

**Review by stevenbal — CHANGES_REQUESTED** [review]
@annashamray @Coperh I'm trying out the CSP changes to fix the redoc schema (with Objects API). But it doesn't seem to work?

```
Content-Security-Policy: The page’s settings blocked a script (script-src-elem) at https://cdn.jsdelivr.net/npm/redoc@latest/bundles/redoc.standalone.js from being executed because it violates the following directive: “script-src 'self' 'unsafe-inline'”
```

Also it's kind of unfortunate we need `'unsafe-inline'` for both script-src and style-src, because having those enabled kind of defeats the purpose of using CSP, but if there's not other way to get redoc to work, than that will have to be like that I guess

**Comment by Coperh** [comment]
@stevenbal Did you add the sidecar to the spectacular settings? 
```
'SWAGGER_UI_DIST': 'SIDECAR',
'SWAGGER_UI_FAVICON_HREF': 'SIDECAR',
'REDOC_DIST': 'SIDECAR',
```

https://drf-spectacular.readthedocs.io/en/latest/faq.html#my-swagger-ui-and-or-redoc-page-is-blank

**Comment by annashamray** [comment]
@stevenbal I agree with you about `'unsafe-inline'`, drf-spectacular suggest using `SpectacularSwaggerSplitView` as a workaround for Swagger UI, but we use redoc and I haven't seen any solution for it

**Comment by annashamray** [comment]
I see a number of issues related to csp in drf-spectacular github (I think this issue is quite promising https://github.com/tfranzel/drf-spectacular/issues/1241 ) but it's unclear if they actually plan to implement it

**Comment by annashamray** [comment]
@stevenbal I've checked it with Objects API, it works for me (after you add `'REDOC_DIST': 'SIDECAR'` to spectacular settings)

**Comment by stevenbal** [comment]
@annashamray ah I didn't check that, in that case all good :+1:

**Pull request merged**